Branch-SDK/Branch-SDK/BNCServerInterface.m

```diff
@@ -121,6 +121,7 @@ - (void)genericHTTPRequest:(NSURLRequest *)request retryNumber:(NSInteger)retryN
         }
     }];
     [task resume];
+    [session finishTasksAndInvalidate];
```

Comment:
It doesn't appear to be necessary, per the [documentation](https://developer.apple.com/library/ios/documentation/Foundation/Reference/NSURLSession_class/#//apple_ref/occ/instm/NSURLSession/finishTasksAndInvalidate): `Once a session is invalidated, new tasks cannot be created in the session, but existing tasks continue until completion.`